extensions/amp-user-notification/validator-amp-user-notification.protoascii

```diff
@@ -31,7 +31,6 @@ tags: {  # <amp-user-notification>
   tag_name: "AMP-USER-NOTIFICATION"
   disallowed_ancestor: "AMP-SIDEBAR"
   requires_extension: "amp-user-notification"
-  requires_extension: "amp-analytics"
   attr_lists: "extended-amp-global"
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-user-notification"
```

Comment:
Can you remove this `spec_url` line as well?

It was only there because this tag didn't know which extension spec to refer to, but now that there is only one to choose, it's unnecessary.